src/onepay-checkout.js

```diff
@@ -18,7 +18,6 @@ let SOCKET_CREDENTIALS_URL = 'https://w7t4h1avwk.execute-api.us-east-2.amazonaws
 let OTT_EXPIRATION = 10; // En minutos
```

Comment:
It makes more sense to have these declarations be ´´´const´´´s since they are constants